configure

```diff
@@ -206,6 +206,7 @@ my @specs =
     'optimize!', 'Optimization (no)',
     'inline!', 'Inlining (yes)',
     'static!', 'Static libraries (no)',
+    'static-runtime!', 'Link to static platform runtime libraries',
```

Comment:
My understanding is that this option should only be used with `--static`. If that is correct, should we document that here or somewhere else?

Reply:
It will only take effect with `--static`.  I think the existing mechanisms for understanding the configure script's behavior, namely using `--verbose`, running `tools/scripts/show_build/config.pl`, and reading the code are sufficient, but some may disagree.